### PR TITLE
Added slack connector topic

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Added a topic for information about using Slack connectors for sink tables.

### CODE REVIEW

File: docs/products/flink/howto/slack-connector.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+5. In the **Table SQL** section, set the connector to **slack** and enter the necessary token as shown in the example below:
+
+::
+
+    CREATE TABLE channel_name (
+    channel_id STRING,
+    message STRING
+    ) WITH (
+        'connector' = 'slack',
+        'token' = 'your_token_here'
```
Comment (ftisiot) [inline docs/products/flink/howto/slack-connector.rst:40]: We should write that the `channel_id` is the channel ID parameter and the token is the token value

**Review by ftisiot — CHANGES_REQUESTED** [review]
Almost good to go, I would add a note to substitute the parameters

File: docs/products/flink/howto/slack-connector.rst (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/slack-connector.rst:40]: Fixed.

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**